lib/pages/chat_draft/draft_chat.dart

```diff
@@ -58,6 +57,8 @@ class DraftChatController extends State<DraftChat>
 
   EmojiPickerType emojiPickerType = EmojiPickerType.keyboard;
 
+  final isSendingNotifier = ValueNotifier(false);
```

Comment:
```suggestion
  final ValueNotifier<bool> isSendingNotifier = ValueNotifier(false);
```

STRONG STRONGGGGGG

Reply:
no need, because it's clear in the right hand side

Reply:
!

<img width="549" alt="Screenshot 2023-08-30 at 16 18 13" src="https://github.com/linagora/twake-on-matrix/assets/99852347/e718a005-559d-4bdf-b93d-6d9b87338dbb">

Reply:
<img width="735" alt="Screenshot 2023-08-30 at 16 25 31" src="https://github.com/linagora/twake-on-matrix/assets/43041967/393fc330-b89c-4ace-a381-b36f173c8cae">

Reply:
agree with Khải